### PR TITLE
Official release of #mathildagame

### PR DESCRIPTION
**Pull request by IBwWG**

### CODE REVIEW

File: src/documents/showcase/Willy and Mathilda's Houseboat Adventure꞉ The Game.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+```
+layout: showcase
+title: "Willy and Mathilda's Houseboat Adventure? The Game"
```
Comment (Gama11) [inline src/documents/showcase/Willy and Mathilda's Houseboat Adventure꞉ The Game.md]: I'm guessing this is supposed to be `:` instead of `?``.

Comment (Gama11) [inline src/documents/showcase/Willy and Mathilda's Houseboat Adventure꞉ The Game.md]: Or, well, that special unicode `:` that is usable in file names anyway.. :D

Comment (IBwWG) [inline src/documents/showcase/Willy and Mathilda's Houseboat Adventure꞉ The Game.md]: That is what was intended, and actually I had copied it from somewhere else.  I guess it didn't survive the copy-paste?  Or is it just not displaying properly?

**Comment by IBwWG** [comment]
There, that seems to show right.  I wonder what I did wrong there.  I prepared it a few months ago though so I simply don't remember anymore.

**Comment by IBwWG** [comment]
Thanks for catching that :)

**Comment by Gama11** [comment]
Hm.. is `: The game` important to include in the title? The title needs 3 lines otherwise, I think this will break the showcase layout:



(this is just a quick test with the Chrome developer tools, that's why the targets aren't correct)

**Comment by IBwWG** [comment]
It was kind of important.  I've seen longer game titles though...maybe the layout could be expanded in the future.  For now I can just change it to #mathildagame if it's a big deal.

**Comment by IBwWG** [comment]
BTW how'd this break the build?

**Comment by Gama11** [comment]
It didn't, the TravisCI setup on this repo is simply broken right now.

I don't know how to fix the layout, so somebody else would have to look into that.
What's wrong with just omitting the `The game` part? Using a hashtag seems a bit silly to me.

**Comment by IBwWG** [comment]
To distinguish it from the original book, if you're searching for it.

**Comment by IBwWG** [comment]
I guess we're small time enough at this point it doesn't matter.

**Comment by Gama11** [comment]
The file names need to be adjusted as well to match the title I think.

**Comment by IBwWG** [comment]
Shoot, won't let me rename the image.  So much for doing things ahead of time.

**Comment by Gama11** [comment]
The windows explorer?

**Comment by IBwWG** [comment]
I know, I just didn't have it cloned anywhere yet.  There, now it's done.

**Comment by Gama11** [comment]
Alright, should be good now. Thanks!

Can't say when this will go live. Can't update the site myself right now since @impaler restructured some things.

**Comment by IBwWG** [comment]
K cool, thanks.

**Pull request merged**